website/docs/en/config/target.mdx

```diff
@@ -10,22 +10,21 @@ The following options are now supported:
 
 | options                      | description                                                                                                                                                                                                                                   |
 | ---------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `'web'`                      | Compile as available in the browser environment (default)                                                                                                                                                                                     |
+| `'webworker'`                | Compile as WebWorker                                                                                                                                                                                                                          |
+| `'browserslist'`             | Infer the ECMAScript version based on the configured browserslist (default if browserslist config is available)                                                                                                                               |
```

Comment:
```suggestion
| `'browserslist'`             | Infer the ES-features version based on the configured browserslist (default if browserslist config is available)                                                                                                                               |
```

The copywriting on webpack doc is

> Infer a platform and the ES-features

Rspack / webpack will try to map browserslist direct to ES-feature https://github.com/web-infra-dev/rspack/blob/main/packages/rspack/src/config/target.js#L24 instead of a certain ECMA version.